thirdparty/maven/Makefile

```diff
@@ -0,0 +1,71 @@
+PKG_ROOT := $(shell pwd)
+
+include $(PKG_ROOT)/../../package.def
+MAVEN_VERSION := 3.3.9
+pvers := $(MAVEN_VERSION)
+pname := maven
+pfile := apache-$(pname)-$(pvers)-src.tar.gz
+psrc_file := $(SRC_DIR)/$(pfile)
+purl := http://archive.apache.org/dist/maven/maven-3/$(pvers)/source/$(pfile)
+zname := zimbra-$(pname)
+zspec := $(pname).spec
+export M2_HOME := build
+
+.PHONY: all getsrc setup build clean pkgadd pkgrm
+
+all: clean getsrc build pkgrm1
+
+# $(@D) == directory part of the target
+getsrc: $(psrc_file)
+$(psrc_file):
+	$(MKDIR) $(@D) && $(CD) $(@D) && $(WGET) $(purl)
+
+pkgadd: pkgadd_$(PKG_EXT)
+
+pkgadd_rpm:
+	$(PKG_EXTRACT) java-1.7.0-openjdk ant
```

Comment:
Oh yes, missed that. I was playing around and forgot to revert that change. Will make it now.